src/store/wikis.js

```diff
@@ -55,9 +55,13 @@ const mutations = {
     const federatedPropertiesSetting = details.public_settings.find(setting => setting.name === 'wikibaseFedPropsEnable')
     const wikibaseFedPropsEnable = federatedPropertiesSetting ? parseInt(federatedPropertiesSetting.value) === 1 : false
 
+    const logoSetting = details.public_settings.find(setting => setting.name === 'wgLogo')
+    const logoUrl = logoSetting ? logoSetting.value : null
+
     state.currentWikiSettings = {
       entityMapping,
-      wikibaseFedPropsEnable
+      wikibaseFedPropsEnable,
+      logoUrl
```

Comment:
I kept naming these with the same name as they are stored in the db to avoid any [confusion](https://github.com/wbstack/api/blob/main/app/Http/Controllers/WikiSettingController.php#L23). entityMapping isn't so maybe we don't care?

Reply:
Keeping these names aligned does seem like a good idea. I'll upload a follow-up, thanks!!